webbpsf/trending.py

```diff
@@ -56,12 +60,18 @@ def wavefront_time_series_plot(opdtable, start_date=None, end_date=None, label_v
 
     rmses = []
 
+    try:
+        rmses = opdtable['rms_wfe']
+    except KeyError:
+        pass
+
     dates = astropy.time.Time(opdtable['date'], format='isot')
     pre_or_post = []
 
     for row in opdtable:
-        full_file_path = os.path.join(webbpsf.utils.get_webbpsf_data_path(), 'MAST_JWST_WSS_OPDs', row['fileName'])
-        rmses.append(fits.getheader(full_file_path, ext=1)['RMS_WFE'])
+        if full_path is False:
+            full_file_path = os.path.join(webbpsf.utils.get_webbpsf_data_path(), 'MAST_JWST_WSS_OPDs', row['fileName'])
+            rmses.append(fits.getheader(full_file_path, ext=1)['RMS_WFE'])
```

Comment:
Did you mean to have the `rmses.append` line inside the `if` statement here? I don't see how this works if `full_path` is set to true?

Reply:
Yeah I can clean this up to check if the column already exists in the opdtable.  TAS has the ability to provide the RMS WFE already in the generation of the table to this method.  But if grabbing from MAST then WebbPSF would open each FITS file to extract the value.